Dorr atlas blog (#399)
brainglobe/brainglobe.github.io: docs/source/blog/dorr-mouse-mri-atlas-added.md

---
blogpost: true
date: November 5, 2025
author: Saarah Hussain
location: London, England
category: brainglobe
language: English
---

# A new mouse MRI atlas been added to BrainGlobe
[Dorr et al. (2008)](https://doi.org/10.1016/j.neuroimage.2008.03.037) created a high-resolution, three-dimensional MRI-based atlas of the adult C57Bl/6J mouse brain, providing  comprehensive anatomical coverage of the cerebrum, cerebellum, and brainstem. 

The atlas was constructed from within-skull, T2-weighted MR images acquired at 32 μm isotropic resolution from forty 12-week-old mice scanned on a 7 T scanner. Following normalisation, registration, and averaging of individual scans, 62 distinct brain structures were manually delineated.

> This atlas is now available through BrainGlobe. We called it `dorr_mouse_mri_32um` and its annotations are closely aligned to the Allen Mouse Brain Atlas.

Reviewer: ```suggestion
This atlas is now available through BrainGlobe as `dorr_mouse_mri_32um`.
```

Using the word "alignment" implies spatial registration. I think best to just remove this.
